src/test/java/edu/ucsb/cs156/team02/controllers/UCSBSubjectControllerTests.java

```diff
@@ -105,4 +97,48 @@ public void api_ucsbsubject_post__user_logged_in() throws Exception {
         String responseString = response.getResponse().getContentAsString();
         assertEquals(expectedJson, responseString);
     }
+
+
+        //test for /get with id
+    @Test
+    public void api_get_id_returns_a_subject_that_exists() throws Exception {
+
+        // arrange
+        UCSBSubject expectedUCSBSubject = UCSBSubject.builder().id(1L).subjectCode("code 1").subjectTranslation("translation 1").deptCode("dept code 1").collegeCode("college code 1").relatedDeptCode("related dept code 1").inactive(true).build();
+        when(ucsbSubjectRepository.findById(eq(1L))).thenReturn(Optional.of(expectedUCSBSubject));
```

Comment:
I think it should be UCSBSubjectRepository
There are a couple of additional instances of this in lines 115, 126, and 134